src/parser.c

```diff
@@ -391,6 +392,25 @@ static void next_token(Parser *p) {
         break;
       }
 
+      // String.
+      // TODO: Decide on delimiters. Should single/double quote be equal? Raw vs escaped? etc.
+      if (p->lastc == '"') {
```

Comment:
This entire thing is literally a switch statement, I’d add a `case '"'` instead of handling this here of all places. ;Þ

Reply:
Yeah I just found where numbers were parsed (the other literal) and added it in the same way

Reply:
Yeah, numbers are handled that way because I didn’t want to add 10 cases haha. I would have used a case range, but MSVC doesn’t support those, so...